roles-static/tsuru_api/tasks/main.yml

```diff
@@ -32,13 +32,17 @@
   wait_for: port={{api_port}} timeout=60
 
 # TODO: handle the case where the user already exists
-- name: Add admin user.
+- name: Add admin user team
   shell: >
     mongo tsuru --eval 'db.teams.update({_id: "admin"}, {_id: "admin"}, {upsert: true})';
     mongo tsuru --eval "db.teams.update({_id: 'admin'}, {\$addToSet: {users: '{{admin_user}}'}})";
```

Comment:
It seems to me that apart from the exit code we see the output of both commands in ansible, in this case enough to understand what might have gone wrong. Here an execution output, result of both commands looks quite readable to me:

changed: [10.128.1.200 -> 10.128.1.22] => {"changed": true, "cmd": "mongo tsuru --eval 'db.teams.update({_id: \"admin\"}, {_id: \"admin\"}, {upsert: true})'; mongo tsuru --eval \"db.teams.update({_id: 'admin'}, {\\$addToSet: {users: 'administrator@gds.tsuru.gov'}})\";", "delta": "0:00:00.095792", "end": "2015-04-28 15:15:46.171138", "rc": 0, "start": "2015-04-28 15:15:46.075346", "stderr": "", "stdout": "MongoDB shell version: 2.6.9\nconnecting to: tsuru\nWriteResult({ \"nMatched\" : 1, \"nUpserted\" : 0, \"nModified\" : 1 })\nMongoDB shell version: 2.6.9\nconnecting to: tsuru\nWriteResult({ \"nMatched\" : 1, \"nUpserted\" : 0, \"nModified\" : 1 })", "warnings": []}